packages/wallets/src/components/SentEmailContent/SentEmailContent.tsx

```diff
@@ -2,22 +2,63 @@ import React, { FC, Fragment, useEffect, useState } from 'react';
 import { Trans } from 'react-i18next';
 import { useCountdown } from 'usehooks-ts';
 import { useActiveWalletAccount, useSettings, useVerifyEmail } from '@deriv/api-v2';
+import {
+    DerivLightIcEmailSentIcon,
+    DerivLightIcFirewallEmailPasskeyIcon,
+    DerivLightIcSpamEmailPasskeyIcon,
+    DerivLightIcTypoEmailPasskeyIcon,
+    DerivLightIcWrongEmailPasskeyIcon,
+} from '@deriv/quill-icons';
 import { PlatformDetails } from '../../features/cfd/constants';
 import useDevice from '../../hooks/useDevice';
-import ChangePassword from '../../public/images/change-password-email.svg';
+import ChangePasswordIcon from '../../public/images/change-password-email.svg';
 import { TPlatforms } from '../../types';
 import { platformPasswordResetRedirectLink } from '../../utils/cfd';
-import { WalletButton } from '../Base';
+import { WalletButton, WalletText } from '../Base';
 import { WalletsActionScreen } from '../WalletsActionScreen';
 import './SentEmailContent.scss';
 
 type SentEmailContentProps = {
     description?: string;
+    isChangePassword?: boolean; // NOTE: This prop is ONLY used for rendering different email icons between either Change Password/Forgot password email modal
     isInvestorPassword?: boolean;
     platform?: TPlatforms.All;
 };
 
-const SentEmailContent: FC<SentEmailContentProps> = ({ description, isInvestorPassword = false, platform }) => {
+// NOTE: key field is not from BE or requirements, its only used for key prop
+const emailReasons = [
+    {
+        content: <Trans defaults='The email is in your spam folder (Sometimes things get lost there).' />,
+        icon: <DerivLightIcSpamEmailPasskeyIcon height='32px' width='32px' />,
+        key: 'EmailInSpamFolder',
+    },
+    {
+        content: (
+            <Trans defaults='You accidentally gave us another email address (Usually a work or a personal one instead of the one you meant).' />
+        ),
+        icon: <DerivLightIcWrongEmailPasskeyIcon height='32px' width='32px' />,
+        key: 'AnotherEmailAddress',
+    },
+    {
+        content: <Trans defaults='The email address you entered had a mistake or typo (happens to the best of us).' />,
+        icon: <DerivLightIcTypoEmailPasskeyIcon height='32px' width='32px' />,
+        key: 'TypoEmailAddress',
+    },
+    {
+        content: (
+            <Trans defaults='We can’t deliver the email to this address (Usually because of firewalls or filtering).' />
+        ),
+        icon: <DerivLightIcFirewallEmailPasskeyIcon height='32px' width='32px' />,
+        key: 'UnableToDeliverEmailAddress',
+    },
+];
+
```

Comment:
These changes are for this:
<img width="1036" alt="Screenshot 2024-05-02 at 12 38 11 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/858d3d04-7810-4c49-b485-5c6507877d96">